app/lib/common/widgets/member_list_entry.dart

```diff
@@ -380,7 +380,7 @@ class MemberListEntry extends ConsumerWidget {
   }
 
   Future<void> changePowerLevel(BuildContext context, WidgetRef ref) async {
-    final newPowerlevel = await showDialog<int?>(
+    final newPowerlevel = await showDialog<int>(
```

Comment:
err, I am pretty sure I already discussed in a previous PR that this is _intentionally_ allowing for `null`. Why does this keep coming up?